src/AssetRegistry/AbstractPricingModule.sol

```diff
@@ -203,4 +198,47 @@ abstract contract PricingModule is Ownable {
 
         assetRiskVars[asset][basecurrency] = riskVars;
     }
+
+    /**
+     * @notice Set the maximum exposure for an asset
+     * @param asset The address of the asset
+     * @param maxExposure The maximum exposure for the asset
+     * @dev This function can only be called by the contract owner. It sets the maximum exposure for the given asset in the exposure mapping.
+     */
+    function setExposureOfAsset(address asset, uint256 maxExposure) public virtual onlyRiskManager {
+        require(maxExposure <= type(uint128).max, "APM_SEA: Max Exposure not in limits");
+        exposure[asset].maxExposure = uint128(maxExposure);
+    }
+
+    /**
+     * @notice Processes the deposit of tokens if it is white-listed
+     * @param asset The address of the asset
+     * param assetId The Id of the asset where applicable
+     * @param amount the amount of tokens
+     * @return success A boolean, indicating if the asset passed as input is whitelisted and the exposure has been updated
+     * @dev Unsafe cast to uint128, meaning it is assumed no more than 10**(20+decimals) tokens can be deposited
+     */
+    function processDeposit(address asset, uint256, uint256 amount)
+        external
+        virtual
+        onlyMainReg
+        returns (bool success)
+    {
+        exposure[asset].exposure += uint128(amount);
+
+        //ToDo: revert or return false? Now it's not consistent with processWithdrawal
```

Comment:
This is the last remark to be cleared out, should we return false or revert.
Should be consistent over withdraw and deposit, and over Vault.sol, MainRegistry.sol and Pricing modules.
Now it is different everywhere.

Reply:
We'll adhere to the 'revert as soon as possible' approach

Reply:
Good, remove comment?